spack/eiger/packages.yaml

```diff
@@ -0,0 +1,216 @@
+packages:
+  all:
+    # default compilers defined by the system
+    # these reflect the current installed PE
+    compiler: [gcc@10.2.0]
+    providers:
+      mpi: [mpich]
+      blas: [cray-libsci]
+      lapack: [cray-libsci]
+      scalapack: [cray-libsci]
+      pkgconfig: [pkg-config]
+    # Permissions for shared instance
+    permissions:
+      write: group
+      group: csstaff
+  autoconf:
+    buildable: false
+    externals:
+    - spec: autoconf@2.69
+      prefix: /usr
+  automake:
+    buildable: false
+    externals:
+    - spec: automake@1.15.1
+      prefix: /usr
+  binutils:
+    variants: +gold~headers+libiberty+nls~plugin
+    externals:
+    - spec: binutils@2.31
+      prefix: /usr
+  bzip2:
+    buildable: false
+    externals:
+    - spec: bzip2@1.0.6
+      prefix: /usr
+  curl:
+    externals:
+    - spec: curl@7.60.0
+      prefix: /usr
+#  go:
+#    buildable: false
+#    externals:
+#    - spec: go@1.11.5
+#      modules:
+#      - go/1.11.5
+#    - spec: go@1.12.9
+#      modules:
+#      - go/1.12.9
+#    - spec: go@1.13.5
+#      modules:
+#      - go/1.13.5
+  python:
+    buildable: false
+    externals:
+    - spec: python@3.8.5
+      modules:
+      - cray-python/3.8.5.0
+  git:
+    buildable: false
+    externals:
+    - spec: git@2.26.2
+      prefix: /usr
+  libtool:
+    externals:
+    - spec: libtool@2.4.6
+      prefix: /usr
+  lz4:
+    externals:
+    - spec: lz4@1.8.0
+      prefix: /usr
+  m4:
+    externals:
+    - spec: m4@1.4.18
+      prefix: /usr
+  mpich:
+    buildable: false
+    externals:
+    - spec: mpich@3.2
+      modules:
+      - cray-mpich/8.0.16
+      - libfabric/1.11.0.0.233
+  perl:
+    buildable: false
+    externals:
+    - spec: perl@5.26.1
+      prefix: /usr
+  pkg-config:
+    externals:
+    - spec: pkg-config@0.29.2
+      prefix: /usr
+#  readline:
+#    externals:
+#    - spec: readline@7.0
+#      prefix: /usr
+  tar:
+    buildable: false
+    externals:
+    - spec: tar@1.30.0
+      prefix: /bin
+  xz:
+    externals:
+    - spec: xz@5.2.3
+      prefix: /usr
+  zlib:
+    externals:
+    - spec: zlib@1.2.11
+      prefix: /usr
+
+  # other libraries
+
+  netlib-scalapack:
+    variants: build_type=Release
+  netlib-lapack:
+    variants: build_type=Release +external-blas+lapacke
+  openblas:
+    variants: +pic +shared threads=openmp ~virtual_machine
+  quantum-espresso:
+    variants: ~elpa +mpi +openmp
+  elpa:
+    variants: +openmp +optflags
+#  cray-libsci:
+#    buildable: false
+#    externals:
+#    - spec: cray-libsci@20.06.1
+#      modules:
+#      - cray-libsci/20.06.1
+  fftw:
```

Comment:
There is now a cray-fftw package. Instead of using fftw, here one could use `cray-fftw`

Reply:
Done!